cli/bench/testdata/deco_apps_requests.json

```diff
@@ -0,0 +1,2905 @@
+[
```

Comment:
considering how long the benchmark takes (~10 seconds), might want to cut this down a bit, not sure

Reply:
I think it's fine for a benchmark